libraries/adaptive-expressions/tests/expressionParser.test.js

```diff
@@ -962,10 +961,6 @@ const generateParseTest = (input, expectedOutput, expectedRefs) => () => {
 }
 
 describe('expression parser functional test', () => {
-    describe("tt", () => {
```

Comment:
Why is this being removed?

Reply:
This test was introduced by mistake before and used as a local test.